examples/mission.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import asyncio
+
+from dronecode_sdk import connect as dronecode_sdk_connect
+from dronecode_sdk import (MissionItem, MissionItems)
+
+drone = dronecode_sdk_connect(host="127.0.0.1")
+
+
+async def run():
+    my_items = []
+    my_items.append(MissionItem(47.398039859999997, 8.5455725400000002, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
```

Comment:
Maybe split this up as as multi-line and add comments after each number

Reply:
Best would be named arguments.

Reply:
Agreed for named arguments, but that's not in the scope of this PR :+1:

Reply:
> Maybe split this up as as multi-line and add comments after each number

Do we _have_ to :sweat_smile:? It looks scary to me when it takes 10 lines. At least here is reads like "a bunch of arguments that should be named arguments" :laughing:.